Weather App/UI/HomeVC/HomeView.swift

```diff
@@ -0,0 +1,266 @@
+
+import Foundation
+import UIKit
+import SnapKit
+
+class HomeView : UIView {
+    /*
+     Dijelim skrin na gornji i donji dio,
+     u donjem dijelu koristim StackView za prikazivanje min i max temperature
+     te jos jedan StackView za prikaz detalja(pressure,humidity,wind)
+     
+     Dodao sam callback za pritisak na 'search' i 'settings' buttone,
+     čime HomeCoordinator zove start() metodu Search ili Settings kordinatora
+     */
+    
+    var searchButtonPressed : (()->Void)?
```

Comment:
Think the name is fine here? And spacing?

Reply:
I think better name is 'onSearchButtonPressed'.